src/lv_draw/lv_draw_img.h

```diff
@@ -120,9 +140,46 @@ bool lv_img_color_format_is_chroma_keyed(lv_img_cf_t cf);
  */
 bool lv_img_color_format_has_alpha(lv_img_cf_t cf);
 
-/**********************
- *      MACROS
- **********************/
+/**
+ * Allocate an image buffer in RAM
+ * @param w width of image
+ * @param h height of image
+ * @param cf a color format (`LV_IMG_CF_...`)
+ * @return an allocated image, or NULL on failure
+ */
+lv_img_dsc_t *lv_img_buf_alloc(lv_coord_t w, lv_coord_t h, lv_img_cf_t cf);
+
+/**
+ * Free an allocated image buffer
+ * @param dsc image buffer to free
+ */
+void lv_img_buf_free(lv_img_dsc_t *dsc);
+
+/**
+ * Get the memory consumption of a raw bitmap, given color format and dimensions.
+ * @param w width
+ * @param h height
+ * @param cf color format
+ * @return size in bytes
+ */
+static inline size_t lv_img_buf_get_img_size(lv_coord_t w, lv_coord_t h, lv_img_cf_t cf)
```

Comment:
Is `size_t` okay here or do I need to change it to `int`?

Reply:
Use `uint32_t` please. We don't use `size_t` right now.

It seems like a long and rarely used function. Is it really worth to inline it?

Reply:
> Is it really worth to inline it?

I like to compartmentalize features like this because it makes them easier to reuse later without having to move code around. I think this kind of function could be reused in another contribution.

Reply:
> I like to compartmentalize features like this because it makes them easier to reuse later without having to move code around. I think this kind of function could be reused in another contribution.

Would it be less reusable without inlining it? I mean still public (not static) just not inline.

Reply:
@kisvegabor It's not really a big deal to me, so I'll move it into `lv_draw_img.c`.

Reply:
Thank you!